cfn.yaml

```diff
@@ -314,19 +314,40 @@ Resources:
       AlarmActions:
         - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:fulfilment-dev
       AlarmName: !Sub High 5XX rate for manage-frontend in ${Stage}
-      MetricName: HTTPCode_Target_5XX_Count
-      Namespace: AWS/ApplicationELB
-      Dimensions:
-        - Name: LoadBalancer
-          Value: !GetAtt ElasticLoadBalancer.LoadBalancerFullName
-        - Name: TargetGroup
-          Value: !GetAtt TargetGroup.TargetGroupFullName
       ComparisonOperator: GreaterThanOrEqualToThreshold
       Threshold: 3
-      Period: 60
       EvaluationPeriods: 2
-      Statistic: Sum
       TreatMissingData: notBreaching
+      Metrics:
+        - Id: total5xx
+          Expression: backend5XX + elb5XX
+          Label: "Count of Backend AND ELB 5XX"
+        - Id: backend5XX
+          MetricStat:
+            Metric:
+              Namespace: AWS/ApplicationELB
+              MetricName: HTTPCode_Target_5XX_Count
+              Dimensions:
+                - Name: LoadBalancer
+                  Value: !GetAtt ElasticLoadBalancer.LoadBalancerFullName
+                - Name: TargetGroup
+                  Value: !GetAtt TargetGroup.TargetGroupFullName
+            Period: 60
+            Stat: Sum
+            Unit: Count
+          ReturnData: false
+        - Id: elb5XX
+          MetricStat:
+            Metric:
+              Namespace: AWS/ApplicationELB
+              MetricName: HTTPCode_ELB_5XX_Count
+              Dimensions:
+                - Name: LoadBalancer
+                  Value: !GetAtt ElasticLoadBalancer.LoadBalancerFullName
+            Period: 60
```

Comment:
the newer style ELB's metric `HTTPCode_ELB_5XX_Count` seems to work with a `Period` of `60` then? - that's good

Reply:
I tested that yesterday:


- I scaled Autoscaling groups down to:
min: 0
desired: 0

Hit `manage-frontend` in code that gave `503` and the alarm was send

I haven't tested if it was accurately 60 but sending the alarm to my email worked